x-pack/plugins/apm/server/lib/apm_telemetry/index.ts

```diff
@@ -21,14 +20,14 @@ import {
   APM_TELEMETRY_SAVED_OBJECT_ID,
   APM_TELEMETRY_SAVED_OBJECT_TYPE,
 } from '../../../common/apm_saved_object_constants';
-import { getApmTelemetryMapping } from '../../../common/apm_telemetry';
```

Comment:
Maybe we can delete that method, and stop maintaining the script `upload-telemetry-data`. It can be deleted on a follow up PR, though. What do you think @elastic/apm-ui?

Reply:
We use `upload-telemetry-data` to test if changes we make in development actually work. Couldn't we keep it have it use `apmSchema`? Not sure why we would want to delete it unless we have a better way to verify that are tasks work in development.

Reply:
@smith sure! If the APM team uses `upload-telemetry-data` for additional purposes other than provide an up-to-date mapping schema to the telemetry cluster. I have nothing against keeping it. That's also why I didn't remove it in this PR. It's your call 🙂

`schema` provides a double validation: TS and Telemetry-Tools about the data matching the structure. I understand if you also want to use the `upload-telemetry-data` for functional tests.

Reply:
@afharo `upload-telemetry-data` is not used to provide mappings to the telemetry cluster; it's only used in development.

Reply:
Got it! I agree then! There's no need to remove it. Sorry for the misunderstanding 😇